.eslintrc-react-test

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
As a side note, although having more than one `.eslintrc` file is technically more correct, I find it much less convenient - particularly with regard to editor integration.  Could an `.eslintrc` file be added that is more generous, i.e. the union of all the available rule-sets?  This would provide a reasonable development experience, and any linting specific to tests/server/etc could still be done through `npm` commands.